src/stubs/sm_attest.c

```diff
@@ -0,0 +1,13 @@
+#include "reactive_stubs_support.h"
+
+uint16_t SM_ENTRY(SM_NAME) __sm_attest(const uint8_t* challenge, size_t len,
+  uint8_t *result)
+{
+    if( !sancus_is_outside_sm(SM_NAME, (void *) challenge, len) ||
+        !sancus_is_outside_sm(SM_NAME, (void *) result, SANCUS_TAG_SIZE) ) {
+      return 1;
```

Comment:
very minor, but it is not better and more consistent to use the `ResultCode enum` from the reactive_stub_support.h here and in the others stubs as well?

Reply:
not sure what would be cleanest: using #defines for the return values, or changing the int return type to an enum, or casting the enum to an int